.github/workflows/pyopenms-wheels.yml

```diff
@@ -8,7 +8,7 @@ on:
   push:
     tags:
       - 'Release*'
-    branches: [ nightly ]
+    branches: [ nightly, fix/paths_actions ]
```

Comment:
If you add the "ci/github_actions" instead of the "fix/paths_actions" is should try to build it in Github actions.

Reply:
ok added it. do you see why the github actions are not triggered for the openms-ci.yml ?